samples/brokers/onvif-video-broker/Program.cs

```diff
@@ -73,10 +73,36 @@ public T Pop()
 		}
 	}
 
+	public class RtspUrlContent {
+		private readonly string rtspUrl;
+		private readonly string maskedRtspUrl;

```

Comment:
what is the purpose of maskedRtspUrl?  it only seems to be written to console, it doesn't seem to serve any functional purpose.

Reply:
and if maskedRtspUrl doesn't serve a functional purpose, i'm not sure why RtspUrlContent class is introduced (rather than just passing a string around)

Reply:
the purpose of maskedRtspUrl is for not showing sensitive data in log

Reply:
It's seems more readable to mask the data ad hoc rather than need to track it as it's own property. Is this data being used enough that optimization is a concern?

Reply:
Change to use a helper class for masking credential in rtsp url.